### PR TITLE
Correct typos in FSharp.Compiler

### PR DESCRIPTION
**Pull request by dungpa**

### CODE REVIEW

**Comment by dsyme** [comment]
Fantastic!    Looks good to me, thanks

File: src/fsharp/tast.fs [hunk 1]
```diff
@@ -2009,7 +2009,7 @@ and
     /// functions, and relates to the F# spec for arity analysis.
     /// For module-defined values, the currying is based 
     /// on the number of lambdas, and in each position the elements are 
-    /// based on attempting to deconstruct the type of the argument as a 
+    /// based on attempting to reconstruct the type of the argument as a
```
Comment (latkin) [inline src/fsharp/tast.fs]: I think this one is actually right

**Comment by latkin** [comment]
LGTM besides the one I commented on

**Comment by dungpa** [comment]
@latkin Done, thanks :-)

**Comment by latkin** [comment]
:+1:

**Pull request closed without merging**